HGGM/Services/EmailSender.cs

```diff
@@ -2,19 +2,73 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Identity.UI.Services;
 using Serilog;
+using MimeKit;
+using MailKit.Net.Smtp;
+using HGGM.Models.Configuration;
+using Microsoft.Extensions.Configuration;
 
 namespace HGGM.Services
 {
     public class EmailSender : IEmailSender
     {
+        private readonly MailConfig _setup;
         private readonly ILogger _log = Log.ForContext<EmailSender>();
 
-        public Task SendEmailAsync(string email, string subject, string htmlMessage)
+        public EmailSender(IConfiguration config)
+        {            
+            this._setup = config.GetSection("EmailSettings").Get<MailConfig>();
+        }
+               
+        public async Task SendEmailAsync(string email, string subject, string message)
         {
-            _log.Warning(new NotImplementedException(), "Email to '{user}': '{subject}'\n{body}", email, subject,
-                htmlMessage);
+            try
+            {
+                //From Address    
+                string fromAddress = _setup.Email;
+                string FromAdressTitle = "HGGM";
+
+                //To Address    
+                string ToAddress = email;
+                string ToAdressTitle = "User";
+                string Subject = subject;
+                string BodyContent = message;
```

Comment:
apply resharper cleanup and follow resharper hints:
local vars start with lowercase letter
use var